src/main/java/org/minions/devfund/benjamin/Squarelotron.java

```diff
@@ -0,0 +1,200 @@
+package org.minions.devfund.benjamin;
+import java.util.Arrays;
+/**
+ * Class Squarelotron refers to a matrix.
+ */
+public class Squarelotron {
+    private int[][] squarelotron;
+    private int size;
+    private static final int MAX_ROTATIONS = 4;
+
+    /**
+     * Squarelotron constructor, fill the object.
+     * @param size squarelotron dimension.
+     */
+    public Squarelotron(int size) {
+        squarelotron = new int[size][size];
+        int number = 1;
+        for (int i = 0; i < size; i++) {
+            for (int j = 0; j < size; j++) {
+                squarelotron[i][j] = number;
+                number++;
+            }
+        }
+        this.size = size;
+    }
+
+    /**
+     * Constructor to assign values to the Squarelotron.
+     *
+     * @param squarelotron value assigned to the Squarelotron matrix.
+     */
+    public Squarelotron(final int[][] squarelotron) {
+        this.squarelotron = copyMatrix(squarelotron);
+        this.size = this.squarelotron.length;
+    }
+
+    /**
+     * Horizontal flip of ring elements.
+     * @param ring number of ring to be flipped.
+     * @return squarelotron object.
+     */
+    public Squarelotron upsideDownFlip(int ring) {
+        return new Squarelotron(verticalFlip(copyMatrix(squarelotron), ring));
+    }
+
+    /**
+     * Transpone ring elements.
+     * @param ring number of ring to be modified.
+     * @return squarelotron object.
+     */
+    public Squarelotron mainDiagonalFlip(int ring) {
+        return new Squarelotron(getTransponedMatrix(ring));
+    }
+
+    /**
+     * Rotate original squarelotron.
+     * @param numberOfTurns number of 90 degrees rotation.
+     */
+    public void rotateRight(int numberOfTurns) {
+        if (numberOfTurns >= 0) {
+            for (int x = 0; x < numberOfTurns % MAX_ROTATIONS; x++) {
+                //int[][] copy = copyMatrix(squarelotronMatrix);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)